sdk/android/azure-android-compat/src/main/java/com/samples/androidcompat/MainActivity.java

```diff
@@ -0,0 +1,14 @@
+package com.samples.androidcompat;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class MainActivity extends AppCompatActivity {
```

Comment:
Does this app actually have a UI? If not, remove the redundant files.

Reply:
We will prioritize moving the samples into instrumented tests then remove the ui elements once that is in place

Reply:
UI has been removed